downstream/modules/automation-hub/proc-set-rhcertified-remote.adoc

```diff
@@ -3,15 +3,15 @@
 [id="proc-set-rhcertified-remote"]
 = Configuring the rh-certified remote repository and syncing Red Hat Certified collections.
 
-You can edit the *rh-certified* remote repository to sync collections from {HubName} hosted on cloud.redhat.com to your local {HubName}.
-By default, your local {HubName} `rh-certified` repository includes the URL for the entire group of Red Hat Certified Collections available on cloud.redhat.com.
+You can edit the *rh-certified* remote repository to sync collections from {HubName} hosted on console.redhat.com to your local {HubName}.
```

Comment:
I commented this earlier but I feel like `console.redhat.com` should be wrapped in ` for monospace font. I checked the SG but don't see a convention for domains. Maybe there should be one.

In any case I'd advocate for putting `console.redhat.com` in an attribute as well.

Reply:
Just wondering if it's doable to have an attribute enclosed in ` characters
- can we add the ` characters in the attributes file ?     
It's not something I've done before.

Reply:
It is possible. Well it was on Satellite docs.

Reply:
If I add an attribute for console.redhat.com, under which heading does it go?

Reply:
I'd put it under "content collections".